### PR TITLE
Adding redirects to SAO ontology

### PR DESCRIPTION
**Pull request by m-markovic**
Redirects to Github repository with documentation created using Widoco

### CODE REVIEW

**Comment by dgarijo** [comment]
@m-markovic,
it looks like the readme is for another ontology.
Also the readme does not have any maintainers. Can you please update it with yourself at least? Thanks!

**Comment by m-markovic** [comment]
Sorry about the C&P error :). Should be ok now.

File: sao/.htaccess [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+RewriteEngine On
+#Change the path to the folder here
+RewriteBase https://rains-uoa.github.io/SAO-Ontology/ 
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ index-en.html [R=303,L]
```
Comment (dgarijo) [inline sao/.htaccess:21]: This is redirecting to a file that does not exist. It should redirect to the URL of your ontology's HTML

File: sao/.htaccess [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+# Rewrite rule to serve JSON-LD content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} application/ld\+json
+RewriteRule ^$ ontology.json [R=303,L]
+
+# Rewrite rule to serve RDF/XML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^$ ontology.xml [R=303,L]
```
Comment (dgarijo) [inline sao/.htaccess:30]: Same comment as above

File: sao/.htaccess [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+# Rewrite rule to serve N-Triples content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} application/n-triples
+RewriteRule ^$ ontology.nt [R=303,L]
```
Comment (dgarijo) [inline sao/.htaccess:34]: Same comment as above

File: sao/.htaccess [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+# Rewrite rule to serve TTL content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} text/turtle [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/turtle 
+RewriteRule ^$ ontology.ttl [R=303,L]
```
Comment (dgarijo) [inline sao/.htaccess:40]: Same as above

File: sao/.htaccess [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+RewriteCond %{HTTP_ACCEPT} .+
+RewriteRule ^$ 406.html [R=406,L]
```
Comment (dgarijo) [inline sao/.htaccess:43]: Same as above

**Comment by dgarijo** [comment]
Sorry, the redirects are not correct. See the comments I left.
Also, in the readme it's fine just to add your name and email/github id.
For an example, see https://github.com/perma-id/w3id.org/tree/master/example

**Comment by dgarijo** [comment]
I hadn't seen rewrite base.

**Pull request merged**